### ISSUE TITLE
Accessibility: https://docs.jupyter.org/en/latest/install.html

### ISSUE THREAD

**Issue by Ibom99** [jupyterhub/outreachy#113]
### What page is this for?

https://docs.jupyter.org/en/latest/install.html

### WAVE accessibility report

![Screenshot (212)](https://user-images.githubusercontent.com/104465078/197320543-045dea33-bca3-4573-ac60-03a374896b14.png)
![Screenshot (213)](https://user-images.githubusercontent.com/104465078/197320546-8cc37148-16a3-47fa-badc-23e7125aa411.png)
https://github.com/jupyterhub/outreachy/issues/38#issuecomment-1273179513

### The HTML element to be changed

```html
1.) Missing Alternative Text(2): 
<img src="https://server.ethicalads.io/proxy/view/3148/2c883a73-b53a-45b9-9563-6c3ca5b37896/" class="ea-pixel">
2.) Empty Form Label:
<label for="toctree-checkbox-1">
<i class="fas fa-chevron-down"></i>
</label>
3.)Contrast Errors(26): All errors are texts that have a lighter blue color styling.
<a class="reference internal nav-link" href="start/index.html">
Try Jupyter
</a>
<a class="current reference internal" href="#">
Install and Use
</a>
<a class="reference internal nav-link" href="#jupyterhub" style="color: rgb(69, 157, 185); background-color: rgb(255, 255, 255);">
JupyterHub
</a>
<a href="https://github.com/jupyter/jupyter/edit/master/docs/source/install.rst">
<i class="fas fa-pencil-alt"></i>

Edit this page
</a>
<a class="reference external" href="https://github.com/jupyter/notebook">
GitHub Repo
</a>
<a class="reference external" href="https://jupyter-notebook.readthedocs.io/en/latest/?badge=latest">
Docs
</a>
<a class="reference internal" href="install/notebook-classic.html">
Installing the classic Jupyter Notebook interface
</a>
<a class="reference internal" href="use/upgrade-notebook.html">
Upgrading Jupyter Notebook
</a>
<a class="reference external" href="https://github.com/jupyterlab/jupyterlab">
GitHub Repo
</a>
<a class="reference external" href="https://jupyterlab.readthedocs.io/en/stable/">
Docs
</a>
<a class="reference external" href="https://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html">
Install instructions
</a>
<a class="reference external" href="https://github.com/jupyterhub/jupyterhub">
GitHub Repo
</a>
<a class="reference external" href="https://jupyterhub.readthedocs.io/en/stable/">
Docs
</a>
<a class="reference external" href="https://jupyterhub.readthedocs.io/en/stable/installation-guide.html">
Install instructions
</a>
<a class="reference external" href="https://github.com/jupyter/jupyter_console">
GitHub Repo
</a>
<a class="reference external" href="https://jupyter-console.readthedocs.io/en/latest/">
Docs and Install instructions
</a>
<a class="reference external" href="https://github.com/jupyter/qtconsole">
GitHub Repo
</a>
<a class="reference external" href="https://qtconsole.readthedocs.io/en/stable/index.html">
Docs
</a>
<a class="reference external" href="https://qtconsole.readthedocs.io/en/stable/installation.html">
Install instructions
</a>
<a class="reference internal" href="install/kernels.html">
Installing Kernels
</a>
<a class="reference internal" href="install/kernels.html#are-any-languages-pre-installed">
Are any languages pre-installed?
</a>
<a class="reference internal" href="install/kernels.html#how-do-i-install-python-2-and-python-3">
How do I install Python 2 and Python 3?
</a>
<a class="reference internal" href="install/kernels.html#how-do-i-install-other-languages-like-r-or-julia">
How do I install other languages like R or Julia?
</a>
<p class="prev-next-title">
Try Jupyter
</p>
<p class="prev-next-title">
Installing the classic Jupyter Notebook interface
</p>

4.) Suspicious Alternitive Links(2):
<img src="_static/jupyter.svg" class="logo__image only-light" alt="Logo image">
<img src="_static/jupyter.svg" class="logo__image only-dark" alt="Logo image">

5.) Redundant Link:
<a href="https://server.ethicalads.io/proxy/click/3070/11dfa795-66fe-4980-b891-56c28b505b71/" rel="nofollow noopener" target="_blank" class="ethical-image-link">
Linked image with alternative text
<img src="https://media.ethicalads.io/media/images/2020/01/readthedocs-logo-fs8_k6Ps9wN.png" alt="Sponsored: Read the Docs">
</a>
REDUNDANT LINK
<a href="https://server.ethicalads.io/proxy/click/3070/11dfa795-66fe-4980-b891-56c28b505b71/" rel="nofollow noopener" target="_blank">
<strong>
Read the Docs for Business:
</strong>
<span>
Private repos with Pull Request previews.
</span>
<strong>
Start your free trial today.
</strong>
</a>
```

### Your proposed HTML (or CSS), after change

```html
1.) Missing Alternative Text: I'll add an `alt` attribute to the `<img>` tag to describe where the image is from.
<img src="https://server.ethicalads.io/proxy/view/3148/2c883a73-b53a-45b9-9563-6c3ca5b37896/" class="ea-pixel" alt="Ad by EthicalAds">
2.)Empty form label: I will add a descriptive title attribute to the `<i></i>`to describe it's function.
<label for="toctree-checkbox-1">
<i class="fas fa-chevron-down" title="Press down arrow for more content."></i>
</label>
3.) I'll select all elements using multiple class selector and style them to a darker blue. 
<a class="reference internal nav-link" href="#jupyterhub" style="color: rgb(3,37,126) ; background-color: rgb(255, 255, 255);">
JupyterHub
</a>

(CSS)
.reference internal nav-link,
.current reference internal,
.reference external,
.reference internal,
.prev-next-title{
color: rgb(3,37,126);
}

4.)Suspicious Alternative Links(2): I will add a more descriptive alternative attribute. 
<img src="_static/jupyter.svg" class="logo__image only-light" alt="Jupyter Logo(light)">
<img src="_static/jupyter.svg" class="logo__image only-dark" alt="Jupyter Logo(dark)">

5.)Redundant Link: (a.)I will remove the link from `<a></a>` that wraps the text and leave it on the <img> because it's suits better or (b.)I will wrap the whole content, image and txt in the link, so where ever in the box is clicked it leads to the URL.

a.)<a href="https://server.ethicalads.io/proxy/click/3070/11dfa795-66fe-4980-b891-56c28b505b71/" rel="nofollow noopener" target="_blank" class="ethical-image-link">
Linked image with alternative text
<img src="https://media.ethicalads.io/media/images/2020/01/readthedocs-logo-fs8_k6Ps9wN.png" alt="Sponsored: Read the Docs">
</a>
REDUNDANT LINK(removed from text)
<strong>
Read the Docs for Business:
</strong>
<span>
Private repos with Pull Request previews.
</span>
<strong>
Start your free trial today.
</strong>

b.)<div class="ethical-content">
<a href="https://server.ethicalads.io/proxy/click/3070/11dfa795-66fe-4980-b891-56c28b505b71/" rel="nofollow noopener" target="_blank" class="ethical-image-link">
Linked image with alternative text
<img src="https://media.ethicalads.io/media/images/2020/01/readthedocs-logo-fs8_k6Ps9wN.png" alt="Sponsored: Read the Docs">
<div class="ethical-text">
Redundant link(removed)
<strong>
Read the Docs for Business:
</strong>
<span>
Private repos with Pull Request previews.
</span>
<strong>
Start your free trial today.
</strong>
</a>
</div>
</div>
```

**Comment by minrk** [comment]
Thanks for the suggestions!

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
